shelley/design-spec/delegation_design_spec.tex

```diff
@@ -699,21 +745,25 @@ \subsubsection{Reward Account Address}
 \item They use account-style accounting, not UTxO-style.
 \item They can not receive funds via transactions. Instead, their
   balance is only increased when rewards are distributed.
-\item They are not associated with a spending key. Instead, there is a
-  one-to-one correspondence between registered staking keys and reward
+\item They do not contain a value address. Instead, there is a
+  one-to-one correspondence between registered stake addresses and reward
```

Comment:
I think we should not use the term "reward account address" anymore. These things are now exactly stake addresses. We can talk about "reward accounts" and "stake addresses" and say that they have a 1:1 correspondence.